.github/workflows/reusable-build-and-test.yml

```
@@ -49,6 +49,12 @@ jobs:
           echo "Using ref $VERSION_VALUE in next steps"
           echo "CKAN_REF=$VERSION_VALUE" >> $GITHUB_ENV
 
+      - name: Read python version from VERSION file
+        run: |
+            PYTHON_VERSION_VALUE=$(cat "ckan-${{ inputs.ckan-major-version }}/PYTHON_VERSION.txt")
+            echo "Using ref $PYTHON_VERSION_VALUE in next steps"
+            echo "PYTHON_REF=$PYTHON_VERSION_VALUE" >> $GITHUB_ENV
```

Let's use `PYTHON_VERSION` as the local var name in this file
```suggestion
            echo "PYTHON_VERSION=$PYTHON_VERSION_VALUE" >> $GITHUB_ENV
```